docs/partners/adopt.md

```diff
@@ -20,126 +20,257 @@ Adopting a pack has several advantages for the partner:
 ![pack example cyren](/doc_imgs/partners/packexample_cyren.png)
 
 ## Process
-The process to Adopt-a-Pack takes under 5 minutes! Follow the step by step instructions and watch the video tutorials below to learn how to submit your adoption pull requests.  
+Choose one of the options below and follow the step by step instructions or watch the video tutorials below to learn how to submit your adoption Pull Requests. 
 
-<strong>Visual Studio Instructions</strong>: If you are creating the pull request from Visual Studio, please follow the below steps. For additional guidance, watch the <a href="https://www.youtube.com/watch?v=9GPkhtRw4Oc">Visual Studio video overview</a>.
-<details>
-<summary>Click here to see the Visual Studio instructions</summary>
+<details><summary><strong>Working on a local clone of Content repo</strong></summary>
```

Comment:
The nested summary does not look so good as it is not clear what belongs to what and ther's no indentation between sections (see attached gif). 
![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/44666568/170583902-b738cd0e-e805-41f6-95bb-ca7f2ce3cffe.gif)

Let's Instead change this to a simple `###` header, like this:
```suggestion
### Option 1: Work on a local clone of the Content repo
These methods assume that you have already [forked the Content repository](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-2-fork-the-github-repo) and [cloned the fork onto your local machine](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-3-clone-the-github-fork-locally).
Choose either one of the methods below to start or complete adoption.

```
And also add an additional header before the [Adopt Using GitHub UI](https://github.com/demisto/content-docs/pull/1103/files#diff-d12af46ecf104a4963eef8a8a31a022f0e69642d0816056c90032653ce98a813R202), such that finally we'll have two headers under the first there will be the `Adopt Using Helper Script` and `Adopt Using a Text Editor/IDE` and under the second there will be the `Adopt Using GitHub UI`

Reply:
Yea, I tried to do nested section but seems that docusaurus fails when using `blockquotes` tag to create a tabbed sub-section. I then used [their documentation ](https://docusaurus.io/docs/markdown-features#details) but as you can see, the nesting doesn't actually render as expected.

Will use h3 sections instead as suggested.